client/src/components/LayerManager.vue

```diff
@@ -229,16 +229,16 @@ export default defineComponent({
       }
 
       if (selectedTrackId !== null) {
-        if ((editingTrack) && !currentFrameIds.includes(selectedTrackId)) {
-          const editTrack = trackMap.get(selectedTrackId);
-
+        if ((editingTrack) && !currentFrameIds.includes(selectedTrackId)
+        && props.camera === selectedCamera.value) {
+          const editTrack = trackMap?.get(selectedTrackId);
+          //Track doesn't exist in the only camera
           if (editTrack === undefined) {
             throw new Error(`trackMap missing trackid ${selectedTrackId}`);
           }
```

Comment:
Should one of the trackStore helper functions (getTrack maybe) be used here?

Reply:
I don't think so, because the helper functions act on a `camMap` not a `trackMap`.  Each `LayerManager` splits of it's own `trackMap` for that camera so it only knows about it's own tracks for drawing.  It's done at the top of the` LayerManager` using the `props.camera`